strings/names/hdemon1.txt

```diff
@@ -0,0 +1,16 @@
+Ma'ar
+Kharn
+Morbius
```

Comment:
PROBABLY don't do this one since it's a pop culture reference

Reply:
That's ok